plugins_src/autouv/wpc_autouv.erl

```diff
@@ -1035,6 +1264,8 @@ handle_command_1({bend,Type,{'ASK',Ask}}, St) ->
                end);
 handle_command_1({bend,{Type,Param}}, St) ->
     wpc_bend:setup({Type,Param}, St);
+handle_command_1({bend,Type,Param}, St) ->
+    wpc_bend:setup({Type,Param}, St);
```

Comment:
This should have been a separate PR, or at least a separate commit :-)
ignore this comment.